Blizzard.IO.RabbitMQ.Tests/Rpc/NetqRpcRabbitClientTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Blizzard.IO.RabbitMQ.Rpc;
+using EasyNetQ;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Tests.Rpc
+{
+    class RequestStub
+    {
+        public int Value { get; set; }
+    }
+
+    class RespondStub
+    {
+        public int Value { get; set; }
+    }
+
+    [TestFixture]
+    public class NetqRpcRabbitClientTests
+    {
+        private Mock<INetqRabbitRpcConnection<Func<Type, object>>> _netqRpcRabbitConnectionMock;
+        private Mock<ILogger> _logger;
+        private Mock<IBus> _busMock;
+
+        private NetqRabbitRpcClient _rpcClient;
+
+        [SetUp]
+        public void Setup()
+        {
+            _logger = new Mock<ILogger>();
+            _netqRpcRabbitConnectionMock = new Mock<INetqRabbitRpcConnection<Func<Type, object>>>();
+            _busMock = new Mock<IBus>();
+            _netqRpcRabbitConnectionMock.Setup(connection => connection.Bus).Returns(_busMock.Object);
+
+            _rpcClient = new NetqRabbitRpcClient(_netqRpcRabbitConnectionMock.Object, _logger.Object);
+        }
+
+        [Test]
+        public void Request_OnValidRequestAndAbstractMessageTypeBus_ShouldReturnValidRespone()
+        {
+            //Arrange
+            RequestStub request = new RequestStub();
+            RespondStub expectedRespond = new RespondStub { Value = 6 };
+            _busMock.Setup(bus => bus.Request<RequestStub, RespondStub>(It.IsAny<RequestStub>())).Returns(expectedRespond);
```

Comment:
Again, since your talking about an exact `request`, you should switch `It.IsAny<RequestStub>()` with `request`

Reply:
I verified it in the assert section.
It is not the Arrange section responsibility to verify this

Reply:
We talked about it